.github/workflows/benchmarks-run.yml

```diff
@@ -29,33 +55,35 @@ jobs:
         run: |
           echo "Running benchmarks..."
           echo "Pallets: ${{github.event.inputs.pallets}}"
+
       - name: Check Out Repo
         uses: actions/checkout@v4
         with:
           ref: ${{github.event.inputs.branch}}
-      - name: Install Required Packages
-        run: |
-          sudo apt-get update
-          sudo apt install -y protobuf-compiler libclang-dev clang cmake
-      - name: Install Rust Toolchain
-        # Match installation steps to CI base docker image
-        run: |
-          curl https://sh.rustup.rs -sSf | HOME=`pwd` bash -s -- -y
-          echo "PATH=`pwd`/.cargo/bin:$PATH" >> $GITHUB_ENV
```

Comment:
Note, the benchmark script runs unit tests at the end, ie `cargo test ...`, so you'd either need to skip that or still have the Rust toolchain installed. Or, could extract that out to a separate post-benchmark job.

I think it's important to still run the tests post-benchmarks, to make sure the updated weights files haven't broken the build.